tests/equality.test.js

```diff
@@ -2,3 +2,28 @@ var describe = require('mocha').describe
 var it = require('mocha').it
 var expect = require('chai').expect
 var strictEqual = require('../equality')
+
+describe('equality', () => {
+    describe('strictEqual', () => {
+        it('return true if two values are same value and type', () => {
+            var sameSame = (5,5);
+            var testSame = strictEqual(sameSame);
```

Comment:
When there are red squiggly lines under your code that means there is something wrong and/or not working. Pay attention to cleaning that up for clean functioning code.  Fantastic jobs on commits but you can break them down even more.  Too many is never too many.